src/models/nebula19-1/assembly.md

```diff
@@ -0,0 +1,401 @@
+
+## Installing 2.5" drives:
+
+The drive cage can hold two 2.5" drives. Drives of 7.5mm, 9mm, or 15mm are supported. Airflow will decrease below recommended levels with drives larger than 15mm.
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to install 2.5" drives:
+
+1. [Remove the top case](#removing-the-top-case).
+2. Unscrew and remove the 2.5" drive cage's cover.
+
+![2.5" SATA drive cage & screws](./img/25-drive-installation.webp)
+
+3. Remove the plastic ring from the top crossbar and slide out four screws for each drive being installed.
+4. Insert two screws into each side of the 2.5" drive, then slide the drive into the drive cage, aligning the rubber mounts on the screws around the metal of the drive cage.
+    - The larger portion of the SATA connector (the power section) should be at the top, while the smaller portion (the data section) should be at the bottom.
+5. If the [optional SATA backplane](#installing-the-sata-backplane) is not present, connect the SATA data and power cables from the bottom of the drive cage.
+    - A cutout in the chassis allows easy access to the bottom of the drive cage.
+
+## Installing the bottom case fan:
+
+The case includes a 140mm Be Quiet! Silent Wings 4 `BQ SIW4-14025-LF-PWM` fan (not installed by default).
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to replace the bottom case fan:
+
+1. [Remove the top case](#removing-the-top-case) and [remove the GPU brace](#installingremoving-the-gpu-brace).
+2. Remove the fan, acryllic spacer, dust filter, and fan screws from the 140mm fan box.
+
+![Botton fan unboxing](./img/bottom-fan-unboxing.webp)
+
+3. Set the machine down on its side.
+4. Line the components up over the fan screw holes.
+    - The dust filter should have the shiny side facing the chassis.
+    - The fan should have the stationary cover facing upwards and the wire facing the front-interior corner of the machine.
+        - The wire can be passed through a cable management hole in the front-interior corner of the machine.
+5. Screw in the bottom fan from the bottom of the chassis.
+
+![Bottom fan screws](./img/bottom-fan-screws.webp)
+
+## Installing side intake fans:
+
+nebula19 supports up to two 120mm side intake fans mounted to the GPU brace. One 120mm Be Quiet! Silent Wings 4 fan (`BQ SIW4-12025-MF-PWM`) is sold as an optional add-on at the time of purchase.
+
+### Steps to install the side intake fans:
+
+1. [Remove the top case](#removing-the-top-case) and [remove the GPU brace](#installingremoving-the-gpu-brace).
+2. Unpack the fan, dust filter, and rubber mounting pins from the fan box.
+
+![Be Quiet! Silent Wings 4 120mm Box](./img/side-fan-unboxing.webp)
+
+3. Place the dust filter and fan in the desired position along the GPU brace.
+    - The shiny side of the dust filter should face the GPU brace.
+    - The spinning side of the fan should face outward, while the stationary label should face inward.
```

Comment:
Is the side fan supposed to be an intake or an exhaust? If it's supposed to be an exhaust, it doesn't really need a dust filter, I wouldn't think. Probably want to clarify this with MechEng.

Reply:
The side fan would be for intake.